### ISSUE TITLE
Unhandled Rejection (Error): missing response

### ISSUE THREAD

**Issue by andresberejnoi** [scaffold-eth/scaffold-eth-challenges#62]
Hello, I recently followed the steps on this repository to run the Ethereum app for the NFT challenge. My problem comes after I run the command `yarn start`.
From the terminal, there is no error message, and the localhost page GUI loads successfully. However, after about 5 seconds, it crashes and shows me the following error:
![Ethereum App Error screenshot](https://user-images.githubusercontent.com/17301409/150993384-036552b6-e334-4eaa-8dce-ab14a814d3d5.png)

Even after the error, the terminal does not show anything wrong, but if I run `yarn deploy` from another window, it fails because it cannot connect to the network localhost. 

It is worth noting that I run this code successfully just about 1 or two weeks ago. I did not change anything. I simply ran the same commands. I'm using Ubuntu 20.04 and Firefox. Additionally, I am not very familiar with internet communication protocols, which seem to be the problem here. 
Is there a simple thing that I should pay attention to? It is confusing to me because this was working before and now it is not. Thanks for the help in advance.

**Comment by codenamejason** [comment]
I am wondering if you are having conflicts with Ubuntu Yarn and yarn 🤔  try uninstalling the Ubuntu Yarn if it is installed and try again. @andresberejnoi

**Comment by andresberejnoi** [comment]
> I am wondering if you are having conflicts with Ubuntu Yarn and yarn 🤔  try uninstalling the Ubuntu Yarn if it is installed and try again. @andresberejnoi 

Thank you, I'll keep exploring that. I did uninstall Ubuntu yarn before. I followed the instructions from the readme on installing yarn with node js. 

It was originally working and then it stopped. I haven't changed anything in my system that I'm aware of. Maybe some automatic Ubuntu update could have messed with my nodeJS yarn installation?

**Comment by codenamejason** [comment]
I used to have similar issues with Windows, I now have a Mac. I am going to turn this into a discussion and see if we cant get some other input. Post the deploy script if you can.

**Comment by codenamejason** [comment]
Did you update the infura id in constants.js by chance? @andresberejnoi

**Comment by andresberejnoi** [comment]
> Did you update the infura id in constants.js by chance? @andresberejnoi

Sorry for the late reply. I believe I did not change that. The only steps I took are the ones listed in the README file. My only guess is that maybe some Ubuntu update somehow messed up with my yarn or node installation between the time I run it for the first time and my last attempt. 

I think I will try to reinstall the whole thing and see what happens. I might also try on Windows if I can't get the server to start.
